### DISCUSSION TITLE
No transparent background color possible for screenshot

### DISCUSSION THREAD

**Discussion by haspelstfs** [pyvista/pyvista#3590] [Q&A]
Hi,

I am having issues to set a transparent background for a screenshot.

I am  trying to generate a Iso-line (slice from a volume object) with transparent background. The screenshot will be used to overlay on an existing image. Unfortunately for me it is impossible to set the Plotter.background color transparent.
methods I have used so far

```py
p = pv.Plotter(border=False, window_size=(int(xBounds * 20000),int(yBounds * 20000))):
pv.global_theme.transparent_background = True
p.set_background(color='#00FFFFFF')
p.background_color=(1,1,1,0)
p.add_mesh(contours_sliced, color="grey", line_width=10, scalars='Z', opacity='linear',use_transparency=True)
```
From the class definition it seems only RGB values without alpha channel are provided?

Best regards,

Philip

**Comment by banesullivan** [comment]
Try setting `pv.global_theme.transparent_background = True` right after importing PyVista and before instantiating the plotter.

**Comment by banesullivan** [comment]
```py
import pyvista as pv
from pyvista import examples

pv.global_theme.transparent_background = True

p = pv.Plotter(border=False)
# p.set_background(color='#00FFFFFF')
p.background_color = (1,1,1,0)
p.add_mesh(examples.load_random_hills(), opacity='linear', use_transparency=True)
p.show(screenshot=...)
```
![download](https://user-images.githubusercontent.com/22067021/200749210-50ea17d0-8d06-40df-9794-4d7bddc7d22c.png)
